webview-ui/src/i18n/locales/en/settings.json

```diff
@@ -623,51 +623,51 @@
 		},
 		"outputLineLimit": {
 			"label": "Terminal output limit",
-			"description": "Maximum number of lines to include in terminal output when executing commands. When exceeded lines will be removed from the middle, saving tokens. <0>Learn more</0>"
+			"description": "Keeps the beginning and end of command output and drops the middle when needed to stay under the limit. Lower to save tokens; raise if you need more middle content. <0>Learn more</0>"
 		},
 		"outputCharacterLimit": {
 			"label": "Terminal character limit",
-			"description": "Maximum number of characters to include in terminal output when executing commands. This limit takes precedence over the line limit to prevent memory issues from extremely long lines. When exceeded, output will be truncated. <0>Learn more</0>"
+			"description": "Hard cap on output size. Keeps the beginning and end and omits the middle if needed. Overrides the line limit to prevent memory issues from very long lines. <0>Learn more</0>"
 		},
 		"shellIntegrationTimeout": {
 			"label": "Terminal shell integration timeout",
-			"description": "Maximum time to wait for shell integration to initialize before executing commands. For users with long shell startup times, this value may need to be increased if you see \"Shell Integration Unavailable\" errors in the terminal. <0>Learn more</0>"
+			"description": "How long to wait for VS Code shell integration before running commands. Increase if you see 'Shell Integration Unavailable' errors. Only applies when using the VS Code terminal. <0>Learn more</0>"
```

Comment:
it is probably still a good idea to include that it is related to long start up times